administrator/components/com_cache/Controller/DisplayController.php

```diff
@@ -55,7 +53,19 @@ public function getQuickiconContent()
 		}
 
 		// Number bytes are returned in format xxx.xx MB
-		$result = strstr(HTMLHelper::_('number.bytes', $size, 'MB', 1, false), '.', true);
+		$bytes = HTMLHelper::_('number.bytes', $size, 'MB', 1);
+		$mb = strstr($bytes, '.', true);
+
+		// No dot found, so just take the converted size
+		if (!$mb)
+		{
+			$mb = $bytes;
+		}
+
+		$result = [];
+
+		$result['amount'] = $mb;
+		$result['sronly'] = Text::sprintf('COM_CACHE_QUICKICON_SRONLY', $mb);
```

Comment:
Think RTL:
To test with (once modified to add the `_N_`)
`$result['sronly'] = Text::sprintf('COM_CACHE_QUICKICON_N_SRONLY', '&#x200E;' . $mb);`
This should display normally a LTR for the value, i.e. always `0.1 MB` and not `MB 0.1`